sigstore-java/src/main/java/dev/sigstore/bundle/BundleFactory.java

```diff
@@ -51,4 +52,17 @@ public static String createBundle(KeylessSigningResult signingResult) {
           "Can't serialize signing result to Sigstore Bundle JSON", e);
     }
   }
+
+  /**
+   * Read a bundle json and convert it back into a keyless signing result for use within this
+   * library
+   *
+   * @param jsonReader a reader to a valid bundle json file
+   * @return the converted signing result object
+   * @throws BundleParseException if all or parts of the bundle were not convertible to library
+   *     types
+   */
+  public static KeylessSigningResult readBundle(Reader jsonReader) throws BundleParseException {
```

Comment:
Why not just take String like the createBundle method produces?

Reply:
Because it could be coming from a file. Using a string is just as easy as doing `new StringReader("my-string")'` which is what happens internally in the library (if you use the `merge(String...)` method)